### ISSUE TITLE
How to add more yolo detection layers for mxnet version yolov3?

### ISSUE THREAD

**Issue by shadyatscu** [dmlc/gluon-cv#769]
Hi, in other version yolov3(e.g pytorch tensorflow or original darknet) it is very easy to add yolo layer by simply changing the cfg file. In mxnet version yolov3, in order to detection those very small objects in my custom dataset, i want to add one more yolo layers to the network. Check the source code in [yolo3.py:](https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/yolo/yolo3.py#L620)
    [stages = [base_net.features[:15], base_net.features[15:24], base_net.features[24:]]
    anchors = [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]]
    strides = [8, 16, 32]

it is very easy to add more yolo masks(anchors) and feature map strides(e.g [[4, 8, 16, 32]
] ), my question is how to change the stages?  thanks a lot!

**Comment by zhreshold** [comment]
stages is a list of size 3, anchors is a list of list of size3, strides is a list of int of size 3.
So to change to 4 stages, make all lists with size 4 accordingly.

**Comment by jordanlui** [comment]
@shadyatscu Could you confirm if you were able to modify the .cfg file? Any chance you can share example of your .cfg file?
I'm following tutorial to train yolov3 with darknet and I'm also trying to change layers or add a layer to improve small object detection.

**Comment by shadyatscu** [comment]
> @shadyatscu Could you confirm if you were able to modify the .cfg file? Any chance you can share example of your .cfg file?
> I'm following tutorial to train yolov3 with darknet and I'm also trying to change layers or add a layer to improve small object detection.

To the best of my knowledge, the mxnet version yolov3 or other detection algorithms don`t have kind of config files to modify parameters or network architectures. So if you wanna chang layers or add a new detection layer, you have to dive into the source code and that`s not so diffcult. I`ve add a new yolo layer for extrem small object detection and thats works well

**Comment by jay-master** [comment]
@shadyatscu Can I ask you something?  Would you let me know how can I edit cfg file to add more detection layer?

**Comment by samaypashine** [comment]
> @shadyatscu Can I ask you something? Would you let me know how can I edit cfg file to add more detection layer?

Did you find out that how can we add more detection layer yet??
